setup.py

```diff
@@ -36,6 +38,7 @@ def find_version(*file_paths):
     url = "https://github.com/bram85/topydo",
     install_requires = [
         'six >= 1.9.0',
+        'arrow >= 0.6.0',
```

Comment:
I've updated to arrow 0.7 on my local machine and everything seems to be working as excepted :)